torch/cuda/__init__.py

```diff
@@ -274,6 +274,15 @@ def _check_capability():
                     old_gpu_warn
```

Comment:
Looks like ``incorrect_binary_warn`` is never used. However its probably more accurate warning